lib/challenge.js

```diff
@@ -30,7 +35,11 @@
   // - a name
   // - an address
 
-var pet_owner;
+var pet_owner = {
+  name:'baxter',
+  address: 'change st, somerville, ma 02145',;
+
+}
```

Comment:
Other than the unnecessary final comma and the incorrect use of semicolon, this is a good object literal. Half credit.